src/views/base.njk

```diff
@@ -0,0 +1,60 @@
+{% block head %}
+  <link rel="stylesheet" href="/stylesheets/govuk-frontend-4.7.0.min.css" />
+{% endblock %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/skip-link/macro.njk" import govukSkipLink %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+
+{% block skipLink %}
```

Comment:
Would be a good addition to have a body tag wrapping the content for semantic html, lines 11 - 60

Reply:
have added in, thank you!

Reply:
Body tag present in DOM